### DISCUSSION TITLE
Japanese channels I cannot ID

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#38] [Show and tell]
http://58.99.33.16:1935/liveedge17/live_108_3.stream/playlist.m3u8

http://125.227.210.37:8939/VideoInput/play.ts <- NHK BS1

http://122.155.180.8:5021 / 14.102.154.164:5021 -> [122.155.180.8.zip](https://github.com/iptv-org/iptv/files/7477074/122.155.180.8.zip) (material that supports HEVC needed) (it contains almost all previously requested channels, have fun) (apparently server name could be thaifw67.s-itv.com)

Managed to find a website linked to link above : http://website.s-itv.com/ (it is ... weird.) / Registrar seems to be AliBaba Computing (AliYun)

Channel listing (thanks to @Central302)

3 = Not Working
95 = CNN International
98 = CNBC Asia
105 = Nat-Geo (w/ chinese subs)
191 = Star Channel2
193 = Star Channel3
205 = WOWOW Live
207 = Nippon TV G+
208 = Movie going in this one.
209 = WOWOW Plus
210 = Sky A Sports
212 = Fuji TV One
213 = Nihon Eiga Senmon Channel
216 = KBS World Japanese
217 = TV Asahi Channel 2
219 = HISTORY Channel Japan
220 = TBS NEWS
221 = TBS Channel 1
223 = TV Asahi Channel 1 (video issues)
224 = WOWOW Cinema
261 = Not Working
263 = Nat-Geo Wild (w/ chinese subs) - Japanese Channel Was Already Shutdown
301 = NHK BS1
302 = BS12
303 = NHK BS Premium
304 = BS Nippon TV
305 = BS-TBS
306 = BS TV Tokyo
307 = BS Fuji TV
308 = BS Asahi
309 = NHK-G TOKYO
310 = NHK-E TOKYO
311 = TOKYO MX1
312 = Nippon TV
313 = TV Asahi
314 = TBS
315 = TV Tokyo
316 = Fuji TV
317 = NHK-G Osaka
318 = NHK-E Osaka
319 = MBS
320 = SUN Television
321 = ABC Asahi Broadcasting System
322 = TV Osaka
323 = Kansai-TV
324 = Yomiuri TV
325 = WOWOW Prime
326 = Star Channel 1
361 = Not Working
396 = TV5Monde Asia (No Sound)
408 = Samurai-TV
409 = GOLF Network
410 = J Sports 3
411 = J Sports 4
412 = Green Channel
413 = GAORA
415 = J Sports 1
416 = J Sports 2
418 = BBC World News
428 = Not Working
440 = Discovery Asia (w/ chinese subs)
457 = Sci-fi Channel (w/ chinese subs)
492 = Nippon TV Plus
493 = Family Gekijo
494 = Kids Station (w/ japanese subs)
495 = Music Japan TV
496 = Discovery Channel Japan
497 = AXN Japan
498 = Animax Japan
499 = Disney Channel Japan

**Comment by Central302** [comment]
http://58.99.33.16:1935/liveedge17/live_108_3.stream/playlist.m3u8 = Looks Like NHK World Premium

**Reply by Central302** [reply]
List of 112.155.180.8
3 = Not Working
95 = CNN International
98 = CNBC Asia
105 = Nat-Geo(Chinese)
191 = Star Channel2
193 = Star Channel3
205 = WOWOW Live
207 = Nippon TV G+
208 = Movie going in this one.
209 = WOWOW Plus
210 = Sky A Sports
212 = Fuji TV One
213 = Commerical Playing
216 = KBS World Japanese
217 = TV Asahi Channel 2
219 = HISTORY Channel(Japanese)
220 = TBS NEWS
221 = TBS Channel 1
223 = TV Asahi Channel 1(Currentry Video Broken)
224 = WOWOW Cinema
261 = Not Working
263 = Nat-Geo Wild(Chinese Sub) - Japanese Channel Was Alreaady Shutdown
301 = NHK BS1
302 = BS12
303 = NHK BS Premium
304 = BS Nippon TV
305 = BS-TBS
306 = BS TV Tokyo
307 = BS Fuji TV
308 = BS Asahi
309 = NHK-G TOKYO
310 = NHK-E TOKYO
311 = TOKYO MX1
312 = Nippon TV
313 = TV Asahi
314 = TBS
315 = TV Tokyo
316 = Fuji TV
317 = NHK-G Osaka
318 = NHK-E Osaka
319 = MBS
320 = SUN Television
321 = ABC Asahi Broadcasting System
322 = TV Osaka
323 = Kansai-TV
324 = Yomiuri TV
325 = WOWOW Prime
326 = Star Channel 1
361 = Not Working
396 = TV5Monde Asia(No Sound)
408 = Samurai-TV
409 = GOLF Network
410 = J Sports 3
411 = J Sports 4
412 = Green Channel
413 = GAORA
415 = J Sports 1
416 = J Sports 2
418 = BBC World News
428 = Not Working
440 = Discovery Asia(Chinese Sub)
457 = Sci-fi Channel(Chinese Sub)
492 = Nippon TV Plus
493 = Family Gekijo
494 = Kids Station(But Have Japanese Captioning)
496 = Music Japan TV(Video ImageBroken)
496 = Discovery Channel Japan
497 = Commercial
498 = Animax Japan
499 = Disney Channel Japan

**Reply by BellezaEmporium** [reply]
> 
> 
> List of 112.155.180.8 3 = Not Working 95 = CNN International 98 = CNBC Asia 105 = Nat-Geo(Chinese) 191 = Star Channel2 193 = Star Channel3 205 = WOWOW Live 207 = Nippon TV G+ 208 = Decode ERROR 209 = WOWOW Plus 210 = Sky A Sports 212 = Fuji TV One 213 = Commerical Playing 216 = KBS World Japanese 217 = TV Asahi Channel 2 219 = HISTORY Channel(Japanese) 220 = TBS NEWS 221 = TBS Channel 1 223 = TV Asahi Channel 1(Currentry Video Broken) 224 = WOWOW Cinema 261 = Not Working 263 = Nat-Geo Wild(Chinese Sub) - Japanese Channel Was Alreaady Shutdown 301 = NHK BS1 302 = BS12 303 = NHK BS Premium 304 = BS Nippon TV 305 = BS-TBS 306 = BS TV Tokyo 307 = BS Fuji TV 308 = BS Asahi 309 = NHK-G TOKYO 310 = NHK-E TOKYO 311 = TOKYO MX1 312 = Nippon TV 313 = TV Asahi 314 = TBS 315 = TV Tokyo 316 = Fuji TV 317 = NHK-G Osaka 318 = NHK-E Osaka 319 = MBS 320 = SUN Television 321 = ABC Asahi Broadcasting System 322 = TV Osaka 323 = Kansai-TV 324 = Yomiuri TV 325 = WOWOW Prime 326 = Star Channel 1 361 = Not Working 396 = TV5Monde Asia(No Sound) 408 = Samurai-TV 409 = GOLF Network 410 = J Sports 3 411 = J Sports 4 412 = Green Channel 413 = GAORA 415 = J Sports 1 416 = J Sports 2 418 = BBC World News 428 = Not Working 440 = Discovery Asia(Chinese Sub) 457 = Sci-fi Channel(Chinese Sub) 492 = Nippon TV Plus 493 = Family Gekijo 494 = Kids Station(But Have Japanese Captioning) 496 = Music Japan TV(Video ImageBroken) 496 = Discovery Channel Japan 497 = Commercial 498 = Animax Japan 499 = Disney Channel Japan

Sensibly similar to what the Redlab host had. Interesting find.

**Reply by Central302** [reply]
495 is "Music Japan TV".

Found: 213 = "Nihon Eiga Senmon Channel",
           497 = "AXN Japan"

**Reply by Central302** [reply]
Update: 208 = "Eisei Gekijo"

**Comment by cloudcase-eu** [comment]
Hi Guys,
sorry to ask a dump question, but how to play any of the channels in that list of 112.155.180.8 ?
Please throw some light.
Thanks

**Reply by BellezaEmporium** [reply]
2 things : HEVC, Proprietary chinese codec.

**Reply by Droyk** [reply]
> 2 things : HEVC, Proprietary chinese codec.

@LaneSh4d0w & where does one get the so-called proprietary Chinese codec?
Please shed some light on it!

**Reply by BellezaEmporium** [reply]
I believe the codec was called VMMS (or something along those lines, been a while since I've found it) and when you type it on Google, the only thing you could find was sheets in Chinese explaining the codec and the program (an obscure one, couldn't find it) you use to load them.

**Comment by adoy-max** [comment]
Hey, is there a currently working link for NHK BS1? The play.ts one in the first post seems to work somewhat but doesn't play properly (the connection is probably too slow).
